feature-group-definitions/grid.yml

```diff
@@ -0,0 +1,30 @@
+spec: https://drafts.csswg.org/css-grid-3/
+caniuse: css-grid
+compat_features:
+  - css.properties.align-items
+  - css.properties.align-self
```

Comment:
(sorry if I keep adding general comments on unrelated lines of code, I just didn't want to add everything in one big comment, to allow for discussion threads).

How do we think about subgrid? Technically, it is *just* a property value you can use with `grid-template-rows/columns`, but it's considered by most developers who know about grid as an entirely separate feature.

In this group, would it just naturally be included by `css.properties.grid-template-*`? Or should be it appear as its own subgroup, or something else?

Reply:
Same comment for masonry, which is also a value that can be used.

Reply:
Sounds great to me.